synfig-studio/src/gui/widgets/widget_soundwave.cpp

```diff
@@ -302,15 +276,14 @@ bool Widget_SoundWave::do_load(const synfig::filesystem::Path& filename)
 {
 #ifndef WITHOUT_MLT	
 	std::string real_filename = Glib::filename_from_utf8(filename.u8string());
-	if (!Widget_SoundWave::is_file_header_valid(real_filename)) {
-		return false;
-	}
 	Mlt::Profile profile;
 	Mlt::Producer *track = new Mlt::Producer(profile, (std::string("avformat:") + real_filename).c_str());
-	if (!track->get_producer() || track->get_length() <= 0) {
+	int sample_rate = track->get_int("audio_sample_rate");
```

Comment:
There is no "audio_sample_rate" rate property in MLT unless you have forked and modified it. You need to look in either `meta.media.attr` properties (do not assume index is 0), or you need to get a frame and get "audio_frequency" propery from the frame.